hardware_interface/src/resource_manager.cpp

```diff
@@ -1398,11 +1398,15 @@ ResourceManager::ResourceManager(
 : resource_storage_(std::make_unique<ResourceStorage>(params.clock, params.logger))
 {
   RCLCPP_WARN_EXPRESSION(
```

Comment:
Should we add a warning for `return_failed_hardware_names_on_return_deactivate_write_cycle_`, too?

Reply:
Good one

Reply:
Done!